face-db-search/app.py

```diff
@@ -3,63 +3,45 @@
 
 
 import os
-import sys
+import click
 
 from jina.flow import Flow
 
-num_docs = os.environ.get('MAX_DOCS', 500)
-image_src = '/tmp/jina/celeb/lfw/**/*.jpg'
 
-
-def config():
-    parallel = 2 if sys.argv[1] == 'index' else 1
-    shards = 8
+def config(task):
```

Comment:
`task` is not used.

Reply:
@nan-wang `parallel = 2 if task == 'index' else 1` There's this condition in the `config` function to set `PARALLEL` env variable.